core-bundle/src/Resources/contao/library/Contao/System.php

```diff
@@ -532,9 +539,21 @@ public static function loadLanguageFile($strName, $strLanguage=null, $blnNoCache
 		$xlfLoader = new XliffFileLoader(static::getContainer()->getParameter('kernel.project_dir'), true);
 		$strCacheDir = static::getContainer()->getParameter('kernel.cache_dir');
 
+		if (null === self::$arrAvailableLanguageFiles)
+		{
+			$availLangFilesPath = Path::join($strCacheDir, 'contao/config/available-language-files.php');
+			self::$arrAvailableLanguageFiles = file_exists($availLangFilesPath) ? include_once $availLangFilesPath : false;
```

Comment:
```suggestion
			self::$arrAvailableLanguageFiles = file_exists($availLangFilesPath) ? include $availLangFilesPath : false;
```

Using `include_once` would store boolean `true` instead of the array if the PHP file got loaded anywhere else in the same request already.